Update the CAGGS docs with the end_offset within the current bucket nuance (#3991)
timescale/docs: use-timescale/continuous-aggregates/refresh-policies.md

    24 hours.

> If you set the `start_offset` or `end_offset` to `NULL`, the range is open-ended

Reviewer: ```suggestion
If you set `start_offset` or `end_offset` to `NULL`, the range is open-ended
```

> and extends to the beginning or end of time. If you set `end_offset` within the current time bucket, and [real-time aggregation][future-watermark] is disabled, the current time bucket is excluded. This is to improve performance: for time-series data that mostly contains writes that occur in the time stamp order, the time buckets that see lots of writes quickly have out-of-date aggregates. You get better performance by excluding the time buckets that are getting a lot of writes.

Reviewer 2: FWIW: another reason is also that you cannot really refresh a partial bucket. You either compute the whole bucket or not at all.

Author: added, thank you

> and extends to the beginning or end of time. If you set `end_offset` within the current time bucket, and [real-time aggregation][future-watermark] is disabled, the current time bucket is excluded. This is because the current bucket is incomplete and can't be refreshed. Excluding the current bucket also improves performance: for time-series data that mostly contains writes that occur in the time stamp order, the time buckets that see lots of writes quickly have out-of-date aggregates. You get better performance by excluding the time buckets that are getting a lot of writes. 

Reviewer: ```suggestion
and extends to the beginning or end of time. If you set `end_offset` within the current time bucket while [real-time aggregation][future-watermark] is disabled, the current time bucket is excluded. Incomplete time buckets cannot be refreshed, this is because the current bucket is incomplete, also time buckets that see lots of writes quickly have out-of-date aggregates. Excluding the current bucket improves performance: time-series data mostly contains writes that occur in time stamp order, you get better performance by excluding the time buckets that are getting a lot of writes. 
```

    24 hours.

If you set the `start_offset` or `end_offset` to `NULL`, the range is open-ended
and extends to the beginning or end of time. 

> If you set `end_offset` within the current time bucket, this bucket is excluded. This is done for two reasons:

Reviewer: ```suggestion
If you set `end_offset` within the current time bucket, this bucket is excluded. This is done for the following reasons:
```
